elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/EmbeddedElasticsearchPolicy.java

```diff
@@ -104,7 +98,7 @@ public static EmbeddedElasticsearchPolicy create() {
    *  }
    * </pre>
    *
-   * @param index index of the index
+   * @param index   index of the index
```

Comment:
Please refrain from aligning parameter descriptions.
It creates lots of unrelated changes, and it is prone to merge conflicts in the future.

See sample reasoning (~ `it creates problems for future maintenance` ): https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment

Reply:
> Please refrain from aligning parameter descriptions.
> It creates lots of unrelated changes, and it is prone to merge conflicts in the future.
> 
> See sample reasoning (~ `it creates problems for future maintenance` ): https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment

well, Let me change it.